Sources/AsyncHTTPClient/ConnectionPool/HTTP1.1/HTTP1ConnectionStateMachine.swift

```diff
@@ -387,7 +387,7 @@ extension HTTP1ConnectionStateMachine.State {
             case .initialized:
                 preconditionFailure("Invalid state")
             case .idle:
-                preconditionFailure("How can we fail a task, if we are idle")
+                preconditionFailure("How can we fail a request, if we are idle")
```

Comment:
This is such an incredible nit, give me a slap if you want

```suggestion
                preconditionFailure("Can't fail an idle request")
```